botorch/acquisition/knowledge_gradient.py

```diff
@@ -428,16 +443,44 @@ def forward(self, X: Tensor) -> Tensor:
         return values.mean(dim=0)
 
 
+class ProjectedValueFunction(AcquisitionFunction):
```

Comment:
Let's make sure names are as descriptive as possible. I added clarifying note in the docstring that this is typically used for value functions.

```suggestion
class ProjectedAcquisitionFunctionFunction(AcquisitionFunction):
```